test/rails_app/config/application.rb

```diff
@@ -33,16 +33,15 @@ class Application < Rails::Application
     # config.assets.enabled = false
 
     config.action_mailer.default_url_options = { host: "localhost", port: 3000 }
-    rails_version = Gem::Version.new(Rails.version)
```

Comment:
Is this removal related to the boolean warning mentioned? If they aren't I think it's better to keep those changes separated in another PR. Then we can take advantage of the warning-free output in the `master` branch too. 😄 
WDYT?

Reply:
yeah, we could have it on master already without this change. As it isn't a big deal and affects only Rails 5.2+, I changed it and pointed to `5-rc` to not change things related to Rails 4.

I'm gonna remove this commit from this branch and commit only the last commit on `5-rc`.